contracts/token/CHAOS.sol

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "../interfaces/IOwnable.sol";
+import "../interfaces/IFarm.sol";
+import "../interfaces/IApproveAndCall.sol";
+import {IWeweReceiver} from "../interfaces/IWeweReceiver.sol";
+
+contract CHAOS is IERC20, IApproveAndCall, Ownable, ReentrancyGuard {
```

Comment:
We are rewriting, I'm not totally confident with this approach on this PR we are rewriting over 0Z implementation, it's like an unnecessary risk from my perspective. I think we can:
1. copy / paste ERC20 from OZ
2. Remove hooks
3. Extends CHAOS from there